DQM/DTMonitorModule/python/dtDQMOfflineSources_Cosmics_cff.py

```diff
@@ -42,11 +46,12 @@
 from DQM.DTMonitorModule.dtTriggerEfficiencyTask_cfi import *
 
 dtSourcesCosmics = cms.Sequence(dtDataIntegrityUnpacker  +
-                                DTDataIntegrityTask +
-                                dtDCSByLumiMonitor + 
-                                dtRunConditionVar + 
-                                dtSegmentAnalysisMonitor +
-                                dtResolutionAnalysisMonitor +
-                                dtEfficiencyMonitor +
-                                dtTriggerEfficiencyMonitor +
-                                dqmInfoDT)
+                         twinMuxStage2Digis +
```

Comment:
Hi @jfernan2 - sorry for not re-reviewing this after your changes - also the producer(s) themselves need to be moved to a path with producers - in this case the L1 sequence I think. Maybe @rekovic  can help.